### PR TITLE
Added support for avro and json schema converters for Sink Connector

### LINKED ISSUE #193: Sink connector needs to be able to support Converters
Sink connector needs to be able to support Converters so that we can support JSON (string without a schema) as we do today, but also JSON with a schema, and AVRO. 

Add a test to ensure that when the connector is configured with avro and json-schema converters it works as expected.

### PR DESCRIPTION
**Pull request by kev-ms**
## Type of PR

- [ ] Documentation changes
- [x] Code changes
- [ ] Test changes
- [ ] CI-CD changes
- [ ] GitHub Template changes

## Purpose of PR
- Convert kafka stored struct back to JSON to store sink record successfully in CosmosDB
- Added supported for `io.confluent.connect.avro.AvroConverter` and `org.apache.kafka.connect.json.JsonConverter` with Schema

## Observability + Testing
- What changes or considerations did you make in relation to observability?
   - Added loggers

- Did you add testing to account for any new or changed work?
   - Added notes to follow up on additional unit/integration testing with mentioned converters

## Review notes
- Should be merged after this PR: #217
- Validate Sink Connector with below configurations:
   - Validate JSON Converter with below values enabled.
      >"key.converter.schemas.enable": "true"
      >"value.converter.schemas.enable": "true"
      >"key.converter.class": "org.apache.kafka.connect.json.JsonConverter"
      >"value.converter.class": "org.apache.kafka.connect.json.JsonConverter"
   - Validate io.confluent.connect.avro.AvroConverter by seeting below values
      >"key.converter.class": "io.confluent.connect.avro.AvroConverter"
      >"value.converter": "io.confluent.connect.avro.AvroConverter"
      >"key.converter.schema.registry.url": "http://schema-registry:8081"
      >"value.converter.schema.registry.url": "http://schema-registry:8081"
      >"key.converter.schemas.enable": "true"
      >"value.converter.schemas.enable": "true"

## Issues Closed or Referenced

- Closes #193 
- References #12 #217 (this references the issue but does not close with PR)

### CODE REVIEW

File: src/main/java/com/microsoft/azure/cosmosdb/kafka/connect/sink/StructToJsonMap.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+                    break;
+                case INT16:
+                    jsonMap.put(fieldName, struct.getInt16(fieldName));
+                    break;
+                case INT64:
+                	if (Timestamp.LOGICAL_NAME.equals(schemaName)) {
+                		jsonMap.put(fieldName, (java.util.Date) struct.get(fieldName));
+                	} else {
+                		jsonMap.put(fieldName, struct.getInt64(fieldName));
+                	}
```
Comment (sivamu) [inline src/main/java/com/microsoft/azure/cosmosdb/kafka/connect/sink/StructToJsonMap.java:43]: Indentation looks a little off here

File: src/main/java/com/microsoft/azure/cosmosdb/kafka/connect/sink/StructToJsonMap.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+import java.util.Map;
+
+import org.apache.kafka.connect.data.Date;
+import org.apache.kafka.connect.data.Field;
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.data.Struct;
+import org.apache.kafka.connect.data.Time;
+import org.apache.kafka.connect.data.Timestamp;
+
+public class StructToJsonMap {
```
Comment (sivamu) [inline src/main/java/com/microsoft/azure/cosmosdb/kafka/connect/sink/StructToJsonMap.java:14]: One linter issue in this class

File: src/main/java/com/microsoft/azure/cosmosdb/kafka/connect/sink/CosmosDBSinkTask.java [hunk 3]
```diff
@@ -70,11 +75,20 @@ public void put(Collection<SinkRecord> records) {
                 logger.debug("Value schema: {}", record.valueSchema());
                 logger.debug("Value.toString(): {}",  record.value() != null ? record.value().toString() : "<null>");
```
Comment (sivamu) [inline src/main/java/com/microsoft/azure/cosmosdb/kafka/connect/sink/CosmosDBSinkTask.java:77]: Linter issue in this method

**Review by sivamu — COMMENTED** [review]
Awesome work here too! Tested with json schema and avro, all working well

Just a few minor linter issues that need addressing.

**Comment by kev-ms** [comment]
Closing this since #230 should cover these changes

**Pull request closed without merging**